src/core/sock/tcp_seg_pool.cpp

```diff
@@ -36,32 +36,25 @@
 
 #define MODULE_NAME "tcp_seg_pool"
 
+enum {
+    TCP_SEG_POOL_COMPENSATION_LEVEL = 16384U,
```

Comment:
agree that configurable batch values would be useful. Probably I will make it as a hidden undocumented parameter. I'd like to avoid overcomplication of parameters, since current PR tries to simplify configuration